docs/website/check_snippets.py

```diff
@@ -0,0 +1,75 @@
+"""
+Load all snippets in snippet folder, check wether they parse with ast and run them
+"""
+import typing as t
+import os
+import ast
+import sys
+
+SNIPPET_DIR = "../snippets"
+
+def get_snippet_list() -> t.List[str]:
```

Comment:
this must be run from `pytest`. all fixtures to drop pipelines, change the segment key, set the path to credentials should be preserved

Reply:
generate a test for each script. it is I think possible to register tests at run time

Reply:
hm, good idea, i will see if i can do it